examples/plot_multi_axes.py

```diff
@@ -0,0 +1,121 @@
+"""
+Test Plots with Multiple Axes in Matplotcheck
+=============================================
+
+Below you will find some examples of how to use MatPlotCheck
+to test Matplotlib objects with more than one axes in Python.
+
+"""
+
+################################################################################
+# Import Packages
+# ---------------
+# To begin, import the libraries that you need.
+
+import matplotcheck.base as pt
+import matplotlib.pyplot as plt
+import pandas as pd
+
+###############################################################################
+# Setup
+# -----
+# First you must create some example data to plot over multiple axes. This
+# data could represent many different types of plots, such as plotting multiple
+# satellite bands, plotting NDVI of multiple study sites over time, and much
+# more.
+# Once you have created your plot, you will create a Matplotcheck
+# ``PlotTester`` object by providing each Matplotlib axes object to a unique
+# ``PlotTester`` object.
+
+# Create example data
+line_1 = pd.DataFrame({"Ax 1 X Vals": [(0), (10)], "Ax 1 Y Vals": [(0), (10)]})
+line_2 = pd.DataFrame({"Ax 2 X Vals": [(10), (0)], "Ax 2 Y Vals": [(0), (10)]})
+
+# Plot the example data on two seperate Axes
+fig, (ax1, ax2) = plt.subplots(1, 2, figsize = (10, 5))
+
+line_1.plot(ax=ax1, x="Ax 1 X Vals", y="Ax 1 Y Vals")
+line_2.plot(ax=ax2, x="Ax 2 X Vals", y="Ax 2 Y Vals")
+
+fig.suptitle("Figure Title")
+
+ax1.set(title="Axes 1", xlabel = "Ax 1 X Vals", ylabel = "Ax 1 Y Vals")
+ax2.set(title="Axes 2", xlabel = "Ax 2 X Vals", ylabel = "Ax 2 Y Vals")
+
+# Create a Matplotcheck PlotTester object for each axes
+plot_tester_1 = pt.PlotTester(ax1)
+plot_tester_2 = pt.PlotTester(ax2)
+
+###############################################################################
+# Test Each Axes
+# --------------
+# Now that you have two ``PlotTester`` objects, one for each axes, you can run
+# tests like you normally would on the plots!
+#
+# As shown below, both axes share the figure title. You can test for the figure
```

Comment:
Change to: "In this example, both axes share the same figure title. You can test..."